packages/queries/src/query.ts

```diff
@@ -19,6 +19,7 @@ export class QueryBuilder<
     TCardinality extends 'one' | 'maybe' | 'many',
     TLazy extends true | undefined,
     TGroupBy extends string[],
+    TName extends string | undefined,
```

Comment:
no need to have the name as a type parameter. This should just be a regular string.

Reply:
The reason being that the name doesn't impact the output of QueryResult